scripts/build/cht-compose.yml.template

```diff
@@ -57,17 +57,11 @@ services:
     image: {{{ repo }}}/cht-haproxy:{{ tag }}
     container_name: {{ haproxy_container_name }}
     hostname: haproxy
-    depends_on:
-      - couchdb.1
-      - couchdb.2
-      - couchdb.3
```

Comment:
I don't think removing this is necessary, is it?

What's the advantage of starting haproxy before the couch images have started?

Reply:
Well, with the configurable containers, we no longer know if these containers (and services) even exist. :(

Reply:
But they're defined in this file above?

Reply:
They're just temporarily there, I'm splitting them out in the next PR, as per our discussion.

Reply:
Coolio.